tools/wasm-build

```diff
@@ -13,12 +13,10 @@ build_folder=${src_dir}/wasm-build
 boost_dir=${EMSDK}/upstream/emscripten/system/include
 pushd ${src_dir}
 
-cp cmake/Wasm.cmake CMakeLists.txt
-
 # cmake
-emcmake cmake -Bwasm-build -DBoost_INCLUDE_DIR=${boost_dir}
+cmake -Bwasm-build -DBoost_INCLUDE_DIR=${boost_dir} -DTrustWalletCore_USE_WASM=ON -DCMAKE_TOOLCHAIN_FILE=${EMSDK}/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake
```

Comment:
Can we omit `TrustWalletCore` prefix? `TrustWalletCore_USE_WASM` looks long and unnecessary

Reply:
Yes, it's just that it use `PROJECT_NAME` internally

Reply:
Yes, `TW` looks good